src/app/containers/Figure/index.jsx

```diff
@@ -15,10 +15,10 @@ const renderCaption = captionValue =>
     <Text text={captionValue} paragraphOverride={Caption} />
   ) : null;
 
-const FigureContainer = ({ src, alt, ratio, copyright, caption }) => (
+const FigureContainer = ({ src, srcSet, alt, ratio, copyright, caption }) => (
   <Figure>
     <ImagePlaceholder ratio={ratio}>
-      <Image alt={alt} src={src} />
+      <Image alt={alt} src={src} srcSet={srcSet} />
```

Comment:
Shouldn't this be `srcset`?

Reply:
React implements element values differently to default HTML, for details see https://reactjs.org/docs/dom-elements.html 

EG: HTML `class` is `className` in React